website/docs/reference/dbt-jinja-functions/config.md

```diff
@@ -63,9 +66,76 @@ __Args__:
 
 The `config.require` function is used to get configurations for a model from the end-user. Configs defined using this function are required, and failure to provide them will result in a compilation error.
 
+:::warning Deprecation warning for meta fallback
+Starting in <Constant name=core" /> v1.10, `config.require()` throw a deprecation warning when it finds a value in `config.meta` instead of at the top level. This fallback behavior will be removed in a future version.
```

Comment:
```suggestion
Starting in <Constant name="core" /> v1.10, `config.require()` throws a deprecation warning when it finds a value in `config.meta` instead of at the top level. This fallback behavior will be removed in a future version.
```